sbt

```diff
@@ -18,10 +18,10 @@ declare -r latest_28="2.8.2"
 
 declare -r buildProps="project/build.properties"
 
-declare -r sbt_launch_ivy_release_repo="https://repo.typesafe.com/typesafe/ivy-releases"
+declare -r sbt_launch_ivy_release_repo="http://repo.typesafe.com/typesafe/ivy-releases"
```

Comment:
I redownloaded the sbt script. Weird change, happy to undo.

Reply:
Yeah I manually changed them to https, I don't know why Paul would want it to be http. We should probably push that change upstream.